Modules/_sqlite/connection.c

```diff
@@ -1866,6 +1866,53 @@ pysqlite_connection_exit_impl(pysqlite_Connection *self, PyObject *exc_type,
     Py_RETURN_FALSE;
 }
 
+/*[clinic input]
+_sqlite3.Connection.setlimit as setlimit
+
+    limit: int
+        The limit category to be set.
+    value: int
+        The new limit. If the new limit is a negative number, the limit is
```

Comment:
Since we add also `getlimit()`, would not be better to make negative value an error?